SU2_CFD/src/fluid/CMutationTCLib.cpp

```diff
@@ -27,46 +27,201 @@
 
 #include "../../include/fluid/CMutationTCLib.hpp"
 
+//#if defined(HAVE_MPP) && !defined(CODI_REVERSE_TYPE) && !defined(CODI_FORWARD_TYPE)
+
 CMutationTCLib::CMutationTCLib(const CConfig* config, unsigned short val_nDim): CNEMOGas(config, val_nDim){
+ 
+  Mutation::MixtureOptions opt(gas_model);
```

Comment:
@CatarinaGarbacz try putting the #if defined(HAVE... inside the constructor, to create a dummy constructor when compiling with Codi.